src/iqstat.cpp

```diff
@@ -276,58 +144,64 @@ main( int argc, char **argv ) {
         usage();
         return 0;
     }
-
     status = getRodsEnv( &myEnv );
     if ( status < 0 ) {
-        rodsLog( LOG_ERROR, "main: getRodsEnv error. status = %d",
-                 status );
-        return 1;
+	// TODO might have to change this for 4.3.0
```

Comment:
I think this comment can be removed as this probably just won't compile in 4.3.0 and doesn't add much value in 4-2-stable (since it's not 4.3 :))

Reply:
yes, no mention of 4.3 in 4-2-stable please.

Reply:
Deleting. I believe this actually might compile in 4.3 assuming the include were changed to <irods/rodsLog.h>.  But, placing the TODO in my notes instead to output directly to stderr

Reply:
squash done